content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,378 @@
+   :align: center
+   :alt: Edit In Progress pop-up window.
+
+- :guilabel:`Name`: the name of the *Stage* in the Kanban pipeline.
+- :guilabel:`Folded in Maintenance Pipe`: when checked, this *Stage's* column will be folded by
+  default in the :guilabel:`Kanban` view type.
+- :guilabel:`Request Confirmed`: when this box is not ticked, and the maintenance request type is
+  set to *Work Center*, no leave is created for the respective work center when a maintenance
+  request is created. If the box is ticked, the work center is automatically blocked for the listed
+  duration, either at the specified date, or as soon as possible, if the work center is unavailable.
+- :guilabel:`Sequence`: the order in the maintenance process in which this *Stage* appears.
+- :guilabel:`Request Done`: if ticked, this box indicates this *Stage* is the final step in the
+  maintenance process. Requests moved to this *Stage* will close.
+
+Once ready, click :guilabel:`Save & Close`. If no changes have been made, click :guilabel:`Cancel`.
+
+List view
+~~~~~~~~~
+
+With the :guilabel:`List` view selected, all open maintenance requests are displayed in a list, with
+information about each request listed in its respective row.
+
+The columns of information displayed in this view type are the following:
+
+- :guilabel:`Subjects`: the name assigned to the maintenance request.
+- :guilabel:`Employee`: the employee who originally created the maintenance request.
+- :guilabel:`Technician`: the *Technician* responsible for the maintenance request.
+- :guilabel:`Category`: the *Category* the equipment being repaired belongs to.
+- :guilabel:`Stage`: the stage of the maintenance process the request is currently in.
+- :guilabel:`Company`: if in a multi-company environment, the company in the database the request is
+  assigned to.
+
+Pivot view
+~~~~~~~~~~
+
+With the :guilabel:`Pivot` view selected, maintenance requests are displayed in a pivot table, and
+can be customized to show different data metrics.
+
+To add more data to the pivot table, click the :guilabel:`Measures` button to reveal a drop-down
+menu. By default, :guilabel:`Count` is selected. Additional options to add to the table are
+:guilabel:`Additional Leaves to Plan Ahead`, :guilabel:`Duration`, and :guilabel:`Repeat Every`.
+
+.. image:: maintenance_calendar/maintenance-calendar-measures-menu.png
+   :align: center
+   :alt: Measures options on Pivot view page.
+
+To the right of the :guilabel:`Measures` button is the :guilabel:`Insert in Spreadsheet` button.
+Clicking this button opens a pop-up window titled :guilabel:`Select a spreadsheet to insert your
+pivot`.
+
+There are two tabs in this pop-up window: :guilabel:`Spreadsheets` and :guilabel:`Dashboards`. Click
+into one of these tabs and select a spreadsheet or dashboard in the database to add this pivot table
+to. Once ready, click :guilabel:`Confirm`. If this table shouldn't be added to a spreadsheet or
+dashboard, click :guilabel:`Cancel`.
```

Comment:
...or they can just exit out of the pop-up, too, right?

Reply:
Right. Suggestion implemented ✅